core/src/main/java/com/graphhopper/reader/dem/CGIARProvider.java

```diff
@@ -80,85 +66,33 @@ public static void main(String[] args) {
 
         // 337.0
         System.out.println(provider.getEle(49.949784, 11.57517));
-        // 453.0
+        // 466.0
         System.out.println(provider.getEle(49.968668, 11.575127));
-        // 447.0
+        // 455.0
         System.out.println(provider.getEle(49.968682, 11.574842));
 
-        // 3131
+        // 3134
         System.out.println(provider.getEle(-22.532854, -65.110474));
 
-        // 123
+        // 120
         System.out.println(provider.getEle(38.065392, -87.099609));
 
         // 1615
         System.out.println(provider.getEle(40, -105.2277023));
         System.out.println(provider.getEle(39.99999999, -105.2277023));
         System.out.println(provider.getEle(39.9999999, -105.2277023));
-        // 1617
+        // 1616
         System.out.println(provider.getEle(39.999999, -105.2277023));
 
         // 0
         System.out.println(provider.getEle(29.840644, -42.890625));
     }
 
-    @Override
-    public void setCalcMean(boolean eleCalcMean) {
-        calcMean = eleCalcMean;
-    }
-
-    void setSleep(long sleep) {
-        this.sleep = sleep;
-    }
-
-    /**
-     * Creating temporary files can take a long time as we need to unpack tiff as well as to fill
-     * our DataAccess object, so this option can be used to disable the default clear mechanism via
-     * specifying 'false'.
-     */
-    public void setAutoRemoveTemporaryFiles(boolean autoRemoveTemporary) {
-        this.autoRemoveTemporary = autoRemoveTemporary;
-    }
-
-    public void setDownloader(Downloader downloader) {
-        this.downloader = downloader;
-    }
-
-    @Override
-    public ElevationProvider setCacheDir(File cacheDir) {
-        if (cacheDir.exists() && !cacheDir.isDirectory())
-            throw new IllegalArgumentException("Cache path has to be a directory");
-        try {
-            this.cacheDir = cacheDir.getCanonicalFile();
-        } catch (IOException ex) {
-            throw new RuntimeException(ex);
-        }
-        return this;
-    }
-
-    protected File getCacheDir() {
-        return cacheDir;
-    }
-
-    @Override
-    public ElevationProvider setBaseURL(String baseUrl) {
-        if (baseUrl == null || baseUrl.isEmpty())
-            throw new IllegalArgumentException("baseUrl cannot be empty");
-
-        this.baseUrl = baseUrl;
-        return this;
-    }
-
-    @Override
-    public ElevationProvider setDAType(DAType daType) {
-        this.daType = daType;
-        return this;
-    }
-
     @Override
     public double getEle(double lat, double lon) {
-        // no data we can avoid the trouble
-        if (lat > 60 || lat < -60)
+        // Return fast, if there is no data available
+        // See https://www2.jpl.nasa.gov/srtm/faq.html
+        if (lat > 60 || lat < -56)
```

Comment:
Don't we need the same 59.999 border like in the multi ele provider?

Reply:
Ah, maybe we need here if lat>=60 and in the multi ele provider `if(lat < 60)`?

Reply:
> Don't we need the same 59.999 border like in the multi ele provider?

Shouldn't really matter, as outside of these borders the values will be 0 anyway. Also I think that the > 59.999 is not necessarily true globally. I tried a couple of routes in Sweden and some showed this effect.

> Ah, maybe we need here if lat>=60 and in the multi ele provider if(lat < 60)?

True this would make more sense. But the values were for lats like 59.999632 and not 60.